Dockerfile

```diff
@@ -1,27 +1,30 @@
-
 FROM ubuntu:bionic
 
 ENV GCSFUSE_REPO gcsfuse-bionic
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
-RUN useradd -u 1234 notroot && \
-    mkdir -p /backups/tmp && \
-    mkdir -p /backups/output && \
-    mkdir -p /mnt/backup-bucket && \
-    chown notroot /backups -R && \
-    chown notroot /mnt/backup-bucket && \
-    apt-get update && apt-get install --yes --no-install-recommends \
-    ca-certificates=20211016~18.04.1 \
-    curl=7.58.0-2ubuntu3.19 \
-    gnupg=2.2.4-1ubuntu1.6 \
-    mydumper=0.9.1-5 \
-    mariadb-client=1:10.1.48-0ubuntu0.18.04.1 \
-  && echo "deb http://packages.cloud.google.com/apt $GCSFUSE_REPO main" \
-    | tee /etc/apt/sources.list.d/gcsfuse.list \
-  && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - \
-  && apt-get update \
-  && apt-get install --yes gcsfuse=0.40.0 --no-install-recommends \
-  && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* 
+
+RUN <<EOF
```

Comment:
I wonder if this fails differently with this EOF approach instead of && ? I think it doesnt matter in this case but I'm not sure

Reply:
The linter hates EOF anyways, so I am reverting it now :)

For context https://www.docker.com/blog/introduction-to-heredocs-in-dockerfiles/